atlasdb-cli/src/test/java/com/palantir/atlasdb/cli/command/TestTimestampCommand.java

```diff
@@ -56,46 +78,120 @@ public TestAtlasDbServices connect(ServicesConfigModule servicesConfigModule) {
         };
     }
 
-    private SingleBackendCliTestRunner makeRunner(String... args) {
-        return new InMemoryTestRunner(TimestampCommand.class, args);
+    @AfterClass
```

Comment:
I have no idea what any of these tests are doing. It's not something I really expect you to clean up - but I also can't really review them.